lib/global.js

```diff
@@ -1,8 +1 @@
-import { announceList } from 'create-torrent'
-import wrtc from 'wrtc'
-
-globalThis.WEBTORRENT_ANNOUNCE = announceList
-  .map(arr => arr[0])
-  .filter(url => url.indexOf('wss://') === 0 || url.indexOf('ws://') === 0)
-
-globalThis.WRTC = wrtc
+console.warn('webtorrent-hybrid: This package is deprecated as of WebTorrent >= 2.3.0 as native WebRTC support has been added to WebTorrent. Please switch to using the "webtorrent" package.')
```

Comment:
```suggestion
console.warn('webtorrent-hybrid: This package is deprecated as of WebTorrent >= 2.3.0 as native WebRTC support has been added to WebTorrent. Please switch to using the "webtorrent" package.')

export * from 'webtorrent'
```

Reply:
whoops, this should be in index, we still need the `WEBTORRENT_ANNOUNCE`

Reply:
`WEBTORRENT_ANNOUNCE` has been re-added. The console.warn must remain in this file to be loaded before the CLI, otherwise the message will be cleared from the console by the CLI redrawing.